src/components/pages/Offerer/OffererDetails/BankInformation/__specs__/BankInformation.spec.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import BankInformation from '../BankInformation'
+import { Offerer } from '../../Offerer'
+
+describe('src | components | pages | Offerer | BankInformation ', () => {
+  const offerer = new Offerer({
+    id: 'AA',
+    name: 'fake offerer name',
+    address: 'fake address',
+    bic: 'ABC',
+    iban: 'DEF',
+  })
+
+  let props
+  beforeEach(() => {
+    props = { offerer }
+  })
+
+  it('should match snapshot', () => {
+    // when
+    const wrapper = shallow(<BankInformation {...props} />)
+
+    // then
+    expect(wrapper).toMatchSnapshot()
+  })
+
+  it('should render instruction block information are not provided', () => {
+    // Given
+    const offererWithoutBI = new Offerer({
+      id: 'AA',
+      name: 'fake offerer name',
+      address: 'fake address',
+      bic: null,
+      iban: null,
+    })
+
+    // when
+    const wrapper = shallow(<BankInformation
+      {...props}
+      offerer={offererWithoutBI}
+                            />)
+
+    // then
+    const bankInstructions = wrapper.find('.bi-instructions')
+    expect(bankInstructions).toHaveLength(1)
+    expect(bankInstructions.text()).toBe(
+      'Renseigner vos coordonnées bancaires pour être remboursé de vos offres éligibles'
+    )
```

Comment:
```suggestion
    wrapper.find({ children: 'Renseigner vos coordonnées bancaires pour être remboursé de vos offres éligibles' })
    expect(bankInstructions).toHaveLength(1)
```
En cherchant par une classe, vous rendez votre test non robuste car l'utilisateur ou le métier ne vois pas cette classe or, l'utilisateur voit ce message et le métier vous a demandé ce message, donc vous testez directement dessus.
Si vous changez le nom de la classe CSS, vous devez changer votre test alors que ça ne devrait pas : c'est un détail d'implémentation.